NineAnimator/Models/Anime Source/animeunity.it/AnimeUnity+Featured.swift

```diff
@@ -21,63 +21,47 @@ import Foundation
-            //div.current-anime>div.row>div a
-            let endpointURL = self.endpointURL
-            let bowl = try SwiftSoup.parse(responseContent)
-            let popularLinkElements = try bowl.select("div.current-anime>div.row>div>div.text-center a")
-            let recentadded = try bowl.select("div.text-center>div.text-center>div a")
-            let recentAnimeLinks = try recentadded.compactMap {
-                aElement -> (a: Element, img: Element)? in
-                if let imageElement = try aElement.select("img").first() {
-                    return (aElement, imageElement)
-                } else { return nil }
-            } .reduce(into: [AnimeLink]()) {
-                container, elements in
-                if let artworkPath = try? elements.img.attr("src"),
-                    let artworkUrl = URL(string: artworkPath, relativeTo: endpointURL),
-                    let animeTitle = (try? elements.a.text())?.components(separatedBy: " - Episodio"),
-                    let animePath = try? elements.a.attr("href"),
-                
-                    let animeUrl = URL(string: animePath, relativeTo: endpointURL) {
-                    // Construct and add anime link
-                    container.append(.init(
-                        title: animeTitle[0],
-                        link: animeUrl,
-                        image: artworkUrl,
-                        source: self
-                    ))
-                }
-            }
-            let popularAnimeLinks = try popularLinkElements.compactMap {
-                aElement -> (a: Element, img: Element)? in
-                if let imageElement = try aElement.select("img").first() {
-                    return (aElement, imageElement)
-                } else { return nil }
-            } .reduce(into: [AnimeLink]()) {
-                container, elements in
-                if let artworkPath = try? elements.img.attr("src"),
-                    let artworkUrl = URL(string: artworkPath, relativeTo: endpointURL),
-                    let animeTitle = try? elements.a.text(),
-                    let animePath = try? elements.a.attr("href"),
-                    let animeUrl = URL(string: animePath, relativeTo: endpointURL) {
-                    // Construct and add anime link
-                    container.append(.init(
-                        title: animeTitle,
-                        link: animeUrl,
-                        image: artworkUrl,
-                        source: self
-                    ))
-                }
+            let data = responseContent
+            let utf8Text = String(data: data, encoding: .utf8) ?? String(decoding: data, as: UTF8.self)
+            let bowl = try SwiftSoup.parse(utf8Text)
+            var encoded = try bowl.select("the-carousel").attr("animes")
+            encoded = encoded.replacingOccurrences(of: "\n", with: "")
+            let data_json = encoded.data(using: .utf8)!
```

Comment:
```suggestion
            let data_json = try encoded.data(using: .utf8).tryUnwrap()
```
Please avoid force casting whenever possible.